packages/flutter_tools/test/general.shard/build_system/depfile_test.dart

```diff
@@ -66,6 +67,23 @@ C:\\a.txt: C:\\b.txt
     FileSystem: () => MemoryFileSystem(style: FileSystemStyle.windows),
   }));
 
+  test('Can escape depfile with windows file paths and spaces in directory names', () => testbed.run(() {
+    final File inputFile = globals.fs.directory(r'Hello Flutter').childFile('a.txt').absolute
+      ..createSync(recursive: true);
+    final File outputFile = globals.fs.directory(r'Hello Flutter').childFile('b.txt').absolute
+      ..createSync();
+    final Depfile depfile = Depfile(<File>[inputFile], <File>[outputFile]);
+    final File outputDepfile = globals.fs.file('depfile');
+    depfile.writeToFile(outputDepfile);
+
+    expect(outputDepfile.readAsStringSync(), contains(r'C:\\Hello\ Flutter\\a.txt'));
+    expect(outputDepfile.readAsStringSync(), contains(r'C:\\Hello\ Flutter\\b.txt'));
+  }, overrides: <Type, Generator>{
+    FileSystem: () => MemoryFileSystem(style: FileSystemStyle.windows),
+    Platform: () => FakePlatform(operatingSystem: 'windows'),
```

Comment:
Test spaces in paths on other platforms, too.

Reply:
Done